src/ast.mli

```diff
@@ -1,4 +1,4 @@
-type program = None
+type program = message list * actor list * func list
```

Comment:
Just compared this with MicroC:
`type program = string list * func_decl list`

it's also a list of functions ... but seems you need string list for statements, expr and such?
https://github.com/mychrisdangelo/microc-withcomments/blob/master/ast.ml (edited)